src/kudu/util/hdr_histogram.h

```diff
@@ -182,6 +186,7 @@ class HdrHistogram {
   int counts_array_length_;
   int bucket_count_;
   int sub_bucket_count_;
+  rw_spinlock histogram_mutex_;
```

Comment:
So, an additional lock is created for every metric created in the system? Wonder if this will create too many locks.

Reply:
Added the refreshMetrics under feature flag. We will revisit this later after we done some perf testing.